### PR TITLE
Revert "64-bit is required for NC26"

### PR DESCRIPTION
**Pull request by come-nc**
Reverts nextcloud/documentation#9274
See https://github.com/nextcloud/server/issues/36072

### CODE REVIEW

File: admin_manual/installation/system_requirements.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ See :doc:`source_installation` for minimum PHP-modules and additional software f
 
 CPU Architecture and OS
 ^^^^^^^^^^^^^^^^^^^^^^^
-A 64-bit CPU and 64-bit OS is required for Nextcloud 26 and higher.
+A 64bit CPU and 64bit OS is required for Nextcloud to run well.
```
Comment (szaimen) [inline admin_manual/installation/system_requirements.rst:42]:
```suggestion
A 64-bit CPU and 64-bit OS is required for Nextcloud to run well.
```

**Comment by szaimen** [comment]
Maybe remove the `(64-bit)` in the table?

**Comment by come-nc** [comment]
> Maybe remove the `(64-bit)` fron the table?

You can open another PR for improvements but the point here is only to revert the 32bit drop information from documentation.

**Review by szaimen — APPROVED** [review]
LGTM

**Pull request merged**